doc/themes.rst

```diff
@@ -182,6 +182,14 @@ Your goal is change the way products are being rendered. In this case you
 create a file in ``templates/mytheme/shuup/front/macros/theme/category.jinja``
 and define the ``{% macro render_products_section() %}`` there with the changes you want.
 
+
+Styles
+------
+
+Theme has a option to specify multiple stylesheets which can be changed from theme
+configuration automatically when defined. See ``shuup/front/templates/shuup/front/base.jinja``
+for example how to use those at the base.jinja.
+
```

Comment:
how about:

Styles
------

Theme can define multiple stylesheets. This helps theme designers to use the 
same base ``.less`` and just overwrite colors from it or make small changes to template. 

See ``shuup/themes/classic_gray/`` for examples on how to define multiple styles and ``shuup/front/templates/shuup/front/base.jinja`` for how to use them in your own `base.jinja`

These styles can then be selected by merchant in theme configuration. 
For Classic Gray, these are defined in ``shuup/themes/classic_gray/theme.py``.